app/proguard-rules.pro

```diff
@@ -81,3 +81,5 @@
 # A reference class used by androidx.concurrent.futures.AbstractFuture$UnsafeAtomicHelper
 # Currently consumed (in depenendecies) by mozilla.components.service.freboard.scheduler
 -dontwarn sun.misc.Unsafe
+# To be addressed in https://github.com/mozilla-mobile/android-components/issues/2900
```

Comment:
nit: this is in the "AndroidX" category. Maybe we should move it up towards the top, in no section.